build.sbt

```diff
@@ -1452,6 +1452,7 @@ lazy val componentsApi = (project in file("components-api"))
         "org.scala-lang.modules"        %% "scala-collection-compat"          % scalaCollectionsCompatV,
         "com.iheart"                    %% "ficus"                            % ficusV,
         "org.springframework"            % "spring-core"                      % springV,
+        "org.springframework"            % "spring-expression"                % springV,
```

Comment:
Do we need the dependency?

Reply:
Not yet. I am splitting bigger PR into smaller ones and this line slipped in :) You can safely ignore it.